Makefile

```diff
@@ -424,22 +427,37 @@ flash_ble: zip
 	nrfutil dfu ble -cd 0 -ic NRF52 -p COM5 -snr 680120179 -f -n $(DICE) -pkg $(ZIP_PATHNAME)
 
 #
-# Factory build
+# Validation build
 #
 
-.PHONY: firmware_factory
-firmware_factory: firmware_fact
+.PHONY: flash_uicr
```

Comment:
Sorry, realizing this only now, but the make commands to build and flash the die in validation mode shouldn't be named like flash_uicr which is too vague, but something like flash_validation for example

Reply:
This is me commenting to show that I fixed my notifications :) Also Olivier is right!

Reply:
I think these are going to go away once we merge the UICR issue branch, but I will rename them for clarity anyhow.